macros/parserMultiAnswer.pl

```diff
@@ -382,8 +383,15 @@ sub perform_check {
   my $inputs = $main::inputs_ref;
   $rh_ans->{isPreview} = $inputs->{previewAnswers} ||
                          ($inputs_{action} && $inputs->{action} =~ m/^Preview/);
+
+  Parser::Context->current(undef,$context);    # change to multi-answser's context
+  my $flags = Value::contextSet($context,$self->cmp_contextFlags($ans)); # save old context flags
+  $context->{answerHash} = $rf_ans;            # attach the answerHash
```

Comment:
```suggestion
  $context->{answerHash} = $rh_ans;            # attach the answerHash
```

Reply:
The patch as is (with the typo on line 389) fixed the issue for me and the prior people who tested it.
The next line which calls `Value::cmp_compare` provides the answerHase `$rh_ans` explictly.
As such, maybe there is no reason to provide the `answerHash` in the context one line 389 and then to remove it on line 392?
At least in the test case provided in https://github.com/openwebwork/pg/issues/632 dropping both those lines from the proposed change seems to still fix the problem.